common/starbase_buildings/hfe_starbase_building_pirate.txt

```diff
@@ -147,11 +148,20 @@ hfe_starbase_building_pirate_minerals = {
 
 hfe_starbase_building_pirate_energy = {
 	icon = GFX_spaceport_module_hydroponic_farms
-	construction_days = 180
+	initial = yes
+	construction_days = 30
 	potential = {
 		exists = owner
 		owner = {
-			is_country_type = hfe_pirate
+			OR = {
+				is_country_type = hfe_pirate
+				is_country_type = dormant_marauders
+				is_country_type = awakened_marauders
+				is_country_type = ruined_marauders
+				is_country_type = marauder_raiders
+				is_country_type = ratlings
+				has_civic = civic_barbaric_despoilers
```

Comment:
Not intended for player

Reply:
The player cannot build them. They can only capture them, and only in special circumstances. Barbaric Despoilers are de facto player pirates.